lib/omnibus/fetchers/net_fetcher.rb

```diff
@@ -215,6 +218,17 @@ def extract
     end
 
     #
+    # The digest type defined in the software definition
+    #
+    # @return [Symbol]
+    #
+    def digest_type
+      DIGESTS.each do |digest|
```

Comment:
Currently, the DIGESTS array is listed with the weakest hash function first.  I'm wondering if we should reverse the order so that the strongest hash function is preferred.